src/main/resources/application.yml

```diff
@@ -141,5 +141,5 @@ aws:
       photos: bb-good-photos
       packages: barter-bay-app-packages
     key:
-      secret: ${S3_SECRET}
-      access: ${S3_ACCESS}
+      secret: dPYZcxWQV2zuK/jQvLh5pkN9QNDS20/46ZVM2Y8n
```

Comment:
Change credentials here. It's very important
And don't forget about it in the future

Reply:
corrected